playbooks/openshift-etcd/private/scaleup.yml

```diff
@@ -70,9 +70,11 @@
                                      | lib_utils_oo_collect('openshift.common.hostname')
                                      | default(none, true) }}"
     openshift_master_etcd_port: "{{ (etcd_client_port | default('2379')) if (groups.oo_etcd_to_config is defined and groups.oo_etcd_to_config) else none }}"
+  handlers:
+  - include: ../../../roles/openshift_master/handlers/main.yml
+    static: yes
   roles:
   - role: openshift_master_facts
   post_tasks:
-  - import_role:
-      name: openshift_master
-      tasks_from: update_etcd_client_urls.yml
+  - include: ../../../roles/openshift_master/tasks/update_etcd_client_urls.yml
```

Comment:
Don't import task files directly from roles.  We use import_role for that.

Also, 'include' is deprecated.

Reply:
I am having this for version <=3.6, I didnt test with master branch. So in these impacted branches we had these dependecies 
https://github.com/openshift/openshift-ansible/blob/release-3.6/roles/openshift_master/meta/main.yml

I am going to decline the PR, I think and I may test with 3.7 because I see that dependencies change was on 3.7